Azure/azure-cli
# {ACR} `az acr build`: Modify `.dockerignore` file paths to be accepted by build

Fixes https://github.com/Azure/azure-cli/issues/17493

In the .dockerignore file, the trailing slash in the listed file paths interferes with the dockerignore build in Windows and Linux environments. By removing that slash, acr build can correctly compare files in its ignore_list to the files that need to be included. This is in parity with Docker's CLI `docker build` command that cleans the file paths listed in dockerignore.

**Testing Guide**
run `acr build --image <image_name> --registry <registry_name> --file <filepath>` on Dockerfile whos .dockerignore contains folders ignored and files within ignored folders that should be included. Be sure that some folders/files have '/' at the end to test this fix.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
